database/src/migrations/20230925103600_create_survey_observations.ts

```diff
@@ -0,0 +1,199 @@
+import { Knex } from 'knex';
+
+/**
+ * @TODO doc
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+
+  ----------------------------------------------------------------------------------------
+  -- Create new survey_observation table
+  ----------------------------------------------------------------------------------------
+
+  SET search_path=biohub;
+
+  CREATE TABLE survey_observation(
+    survey_observation_id                   integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    survey_id                               integer           NOT NULL,
+    wldtaxonomic_units_id                   integer,
+    latitude                                float             NOT NULL,
```

Comment:
I think check constraints on lat / lon would make sense, the range is well known and is never gonna change.

Reply:
Great point. I did some research and settled on the Postgres [numeric](https://www.postgresql.org/docs/current/datatype-numeric.html) type, which gives you custom precision. I'm giving it 7 points of decimal precision.

Reply:
I meant to add check constraints in the form of something like CHECK(latitude >= -90 AND latitude <= 90). Specifying the level of precision also makes sense but I don't think it accomplishes the same thing.

Reply:
I don't think this kind of audit is needed. The `latitude` and `longitude` fields really just reflect the raw data entered in the form, with enough precision to reasonably accept any valid lat/long. Without any requirements associated with how these fields are going to be used in the future, adding constraints on them now would probably be overkill. 

To that end, if we do choose to query these points or need to otherwise validate them in the future, we will probably retain the raw `latitude` and `longitude`, and make an additional column that stores them as a `geography`-like datatype that is design specifically for lat/long points.

Reply:
Well the form should probably be catching this way before it gets to the database, but 99 latitude is just not a correct value and can easily be rejected here, the CHECK constraint can even be added inline with the column declaration. But if you think the requirements on how these columns are going to be used are that uncertain then it's fine to go without.

Reply:
I think the issue is not that it's too unweildy to validate these by the database, but that it's not appropriate for the database to throw an exception for recieving a bogus latitude/longitude, since these columns by design do not need to be valid.